src/map/packets_struct.hpp

```diff
@@ -5975,6 +5975,18 @@ struct PACKET_CZ_ADVENTURER_AGENCY_JOIN_RESULT {
 DEFINE_PACKET_HEADER(CZ_ADVENTURER_AGENCY_JOIN_RESULT, 0x0af8);
 #endif  // PACKETVER_MAIN_NUM >= 20191218 || PACKETVER_RE_NUM >= 20191211 || PACKETVER_ZERO_NUM >= 20191224
 
+struct CZ_REQ_STYLE_CHANGE3_SUB {
```

Comment:
Please move this to packets.hpp
This is the Hercules file and should not be modified.